example/chat.go

```diff
@@ -0,0 +1,232 @@
+// +build ignore
+
+// This is a small chat client
+
+package main
+
+import (
+	"bufio"
+	"encoding/json"
+	"flag"
+	"fmt"
+	"io/ioutil"
+	"net"
+	"net/http"
+	"net/url"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"runtime"
+	"strconv"
+	"time"
+
+	"github.com/andreas/podio-go"
+)
+
+var (
+	podioClient string
+	podioSecret string
+
+	defaultCacheFile = filepath.Join(os.Getenv("HOME"), ".chat-cli-request-token")
+
+	cacheFile = flag.String("cachefile", defaultCacheFile, "Authentication token cache file")
+)
+
+func main() {
+	flag.Parse()
+
+	podioClient = envDefault("PODIO_CLIENT", "chatcli")
+	podioSecret = envDefault("PODIO_SECRET", "4qCaud5yZTt56w6WWbsKp1ldoq0egbEqzTuq7kIU6X6IKy9f9Gjp4K9M9zttXJul")
```

Comment:
Should be fine - these tokens don't have any extra magic - just thought it would be nice with an example that worked out-of-the-box.